### PR TITLE
Adds the Caravan-Class

### PR DESCRIPTION
**Pull request by AquillaF**
Adds the Caravan-Class Modular (but not really) ship.

## About The Pull Request

Adds the Caravan-Class, a ship meant for a smallish crew of 6 people.

Lore wise its meant as a modular ship. With each

 section being able to be changed out to use the user's needs. This configuration is meant as a independent general purpose ship.

Caravan

## Why It's Good For The Game

More variety in ships is always a nice thing.

## Changelog
:cl:
add: Caravan-Class Modular Ship
/:cl:

### CODE REVIEW

File: _maps/shuttles/shiptest/IndiCaravanTypeA.dmm [hunk 1]
```diff
@@ -0,0 +1,3448 @@
...
+/obj/effect/landmark/start/scientist,
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4{
+	dir = 5
+	},
+/turf/open/floor/plasteel/white,
+/area/ship/science)
+"kb" = (
+/obj/machinery/atmospherics/components/trinary/filter/flipped/on/layer4{
+	dir = 4;
+	filter_type = "n2
```
Comment (Vexylius) [inline _maps/shuttles/shiptest/IndiCaravanTypeA.dmm]: missing "

File: _maps/configs/IndiCaravanTypeA.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+	"map_name": "Caravan-class Modular ship",
+	"map_short_name": "Caravan-class",
+	"map_path": "_maps/shuttles/shiptest/IndiCaravanTypeA.dmm",
```
Comment (Apogee-dev) [inline _maps/configs/IndiCaravanTypeA.json:4]: "prefix": "ISV",
"namelists": insert namelists of choice here, recommend GENERAL, SPACE at minimum

File: _maps/configs/IndiCaravanTypeA.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+	{
+		"Captain": 
+		{
+			"outfit": "/datum/outfit/job/captain/western",
+			"officer": true,
+			"slots": 1
+		},
+		"Scientist": 1,
+		"Medical Doctor": 1,
+		"Atmospheric Technician": 1,
```
Comment (Apogee-dev) [inline _maps/configs/IndiCaravanTypeA.json]: you may want to use custom outfits and titles for these three specialist jobs, as the defaults for all three are rather NT-themed. Even if you stick to the default outfits a custom title still adds some useful flavor.

File: _maps/configs/IndiCaravanTypeA.json (hunk 2 above)
Comment (AquillaF) [reply _maps/configs/IndiCaravanTypeA.json:4]: I knew I was forgetting something.

**Review by Apogee-dev — APPROVED** [review]

**Comment by Apogee-dev** [comment]
I've just realized that the ship's pet does not have a custom name. This is a critical oversight and must be rectified at once

**Comment by AquillaF** [comment]
oh yeah, you are right.I cant believe I forgot that.

**Comment by AquillaF** [comment]
New cat name just dropped, Félicette! The first and only cat to have been in space!

**Comment by Latentish** [comment]
While I like this ship, I do feel like it suffers from a lot of squares and boxes. Is there a way you could possibly alter it's outer hull/silhouette to help remedy that? If not I suppose it's more of a minor issue with this ship.

**Comment by Apogee-dev** [comment]
> While I like this ship, I do feel like it suffers from a lot of squares and boxes. Is there a way you could possibly alter it's outer hull/silhouette to help remedy that? If not I suppose it's more of a minor issue with this ship.

I don’t see any problem with the boxiness, especially considering it’s supposed to be a modular, industrial ship. The overall layout still has enough variance to maintain interest and stuff like the gaps and catwalks going on even if it’s made of boxes put together.

**Comment by AquillaF** [comment]
I did do a small test with making it a bit less r e c t a n g u l a r for the forward module. I'm not sure how much I like it though (also testing how wall girders look rather then the lattice + railing).

Caravan2

**Pull request merged**